web/src/search_suggestion.ts

```diff
@@ -482,7 +526,7 @@ function get_special_filter_suggestions(last, terms, suggestions) {
 
     const last_string = Filter.unparse([last]).toLowerCase();
     suggestions = suggestions.filter((s) => {
-        if (match_criteria(terms, s.invalid)) {
+        if (match_criteria(terms, s.invalid!)) {
```

Comment:
I think it would be nicer to, instead of this, have `get_special_filter_suggestions` enforce that it gets suggestions with `invalid` set, like this: `suggestions: (Suggestion & {invalid: InvalidItem[]})[],` (or maybe you can find a slightly cleaner way to do it)

Reply:
Surely we can do this.
Will make changes on this one.